### ISSUE TITLE
min() and max() do not support strings

### ISSUE THREAD

**Issue by JennaSys** [TranscryptOrg/Transcrypt#829]
I didn't see an exception listed in the [docs](https://www.transcrypt.org/docs/html/differences_cpython.html), but the `min()` and `max()` translations are currently using the JS `Math.min()` and `Math.max()` which only support numbers, whereas the Python version also supports string types in the iterable.

```javascript
// Maximum of n numbers
export function max (nrOrSeq) {
    return arguments.length == 1 ? Math.max (...nrOrSeq) : Math.max (...arguments);       
};

// Minimum of n numbers
export function min (nrOrSeq) {
    return arguments.length == 1 ? Math.min (...nrOrSeq) : Math.min (...arguments);       
};
```
If used with an iterable containing strings, the translated max/min returns `NaN`. There are of course workarounds, but this should work the same as CPython or be a documented exception.

**Comment by JennaSys** [comment]
Also consider issue #305 for supporting a key function.

**Comment by JennaSys** [comment]
Added in dev_3.9.3 branch to be released in v3.9.3

**Comment by JennaSys** [comment]
Merged to master on PR #876
